src/sdk/object_sdk.js

```diff
@@ -1107,8 +1109,20 @@ class ObjectSDK {
         this._check_is_readonly_namespace(ns);
         return ns.put_object_acl(params, this);
     }
+
+    is_nsfs_containerized_user_anonymous(token) {
+        return !token && !process.env.NC_NSFS_NO_DB_ENV;
+    }
+
+    is_nsfs_non_containerized_user_anonymous(token) {
+        return !token && process.env.NC_NSFS_NO_DB_ENV;
+    }
 }
 
-module.exports = ObjectSDK;
-module.exports.account_cache = account_cache;
-module.exports.dn_cache = dn_cache;
+// EXPORT
+module.exports = {
+    ObjectSDK,
+    anonymous_access_key: anonymous_access_key,
+    account_cache: account_cache,
+    dn_cache: dn_cache,
+};
```

Comment:
Prefer to use this form to exports multiple things:
```
exports.ObjectSDK = ObjectSDK;
exports.anonymous_access_key = anonymous_access_key;
```
The rational is that replacing the exports object is not safe for callers that use const object_sdk = require('object_sdk') and then later on try to use object_sdk.ObjectSDK as the original exports object is replaced and not used. This is a bit anoying but is more robust.